scripts/github-actions/get-modules-metadata.js

```diff
@@ -0,0 +1,40 @@
+async function getModulesMetadata({ require, core }) {
```

Comment:
Mind adding a [JSDoc ](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) comment to annotate the types of parameters?

Reply:
Sure, updated here:
https://github.com/Azure/bicep-registry-modules/commit/20d412d9da3b54faf759004a8783918aee430fe4

Thanks!